optimade/server/data/providers.json

```
@@ -1 +1 @@
-../../../providers/src/links/v1/providers.json
\ No newline at end of file
+/home/kwibus/miniconda3/lib/python3.7/site-packages/optimade/server/data/providers.json
```

```suggestion
../../../providers/src/links/v1/providers.json
```

Not sure how this happened, but this is breaking some of your CI tests

---

That is strange, I can't remember adding that file to a git commit. I'll look into it.

---

On my machine, providers.json was located in a different folder. And I had changed the link to point to this location.

I have checked all the commands in my terminal, but providers.json was never mentioned in any of my git commands or their output. So I have no idea how the modified link ended up in the repository. 
I have now used the original link and placed a copy of providers.json in this location.

---

In order to get the providers file into the correct place you need to init the git submodule that is part if this repo (`git submodule init; git submodule update`). This is primarily if you wanted to actually use it for hosting; it's not really a big for development, unless you really want all the tests to pass locally.

---

It's necessary for development to not run into this issue ;) But yeah, otherwise it should all be good.

---

Thanks for pointing this out.  I'll add a line to the instructions for the full development installation, so future developers do not run into this issue.